CollaboratedHRM/Pages/HomePage.cs

```diff
@@ -0,0 +1,29 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CollaboratedHRM.Pages
+{
+    public class HomePage
+    {
+
+        private IWebDriver _driver;
+
+        public HomePage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+        private IWebElement LoginName() 
+        {
+            return _driver.FindElement(By.XPath("//p[@class='oxd-userdropdown-name']"));
+        }
+        public void GetUsername() 
+        {
+            String confirmLoginName = LoginName().Text;
+            StringAssert.Contains("Mojisola Otusheso", confirmLoginName);
```

Comment:
Do not assert on user's name as it has a tendency of changing after several run. Try asserting on some other things on the landing page and make more than one assertions